requirements.txt

```diff
@@ -1 +1 @@
-coala==0.3.0
+coala~=0.6.0
```

Comment:
I normally use `>=` for coala version.

Btw, `>=0.5` would be better as it does work with 0.5

Note: This file isnt really used anywhere in the plugin